Cargo.toml

```diff
@@ -4,6 +4,8 @@ members = ["config", "consensus", "crypto", "dag", "executor", "examples", "netw
 [profile.release]
 codegen-units = 1
 lto = true
+# Important for performance/memory profiling
```

Comment:
Cargo now allows [custom profiles](https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles). How about creating a benchmark profile with debug (e.g. "release-profiled"), which would contain the option and edit `CommandMaker` to do `cargo profile --profile release-profiled`

Reply:
Sure.  Although it doesn't seem like profiles enable feature selection, unfortunately.